pkg/runtime/interfaces/application_configuration.go

```diff
@@ -301,7 +301,18 @@ type SignedURL struct {
 	SigningPrincipal string `json:"signingPrincipal"`
 }
 
-// This configuration handles all requests to get remote data such as execution inputs & outputs.
+type InlineEventDataPolicy = string
+
+const (
+	// InlineEventDataPolicyOffload specifies that inline execution event data (e.g. outputs) should be offloaded to the
+	// configured cloud blob store.
+	InlineEventDataPolicyOffload = "offload"
```

Comment:
```suggestion
	InlineEventDataPolicyOffload InlineEventDataPolicy = iota
```

Reply:
how do you users specify this in the config if we're using iota?

Reply:
they can still say `offload` and `inline`... if you generate enumer that generates json marshal and unmarshal and stdlib/config find that automatically and use it....

Reply:
ohhh I see, thanks. done!